src/components/subheader/subheader.js

```diff
@@ -78,6 +78,12 @@ function MdSubheaderDirective($mdSticky, $compile, $mdTheming, $mdUtil) {
         return angular.element(el[0].querySelector('.md-subheader-content'));
       }
 
+      // Set the ARIA attributes on the original element since it keeps it's original place in
+      // the DOM, whereas the clones are in reverse order. Should be done after the outerHTML,
+      // in order to avoid having multiple element be marked as headers.
+      attr.$set('role', 'heading');
+      attr.$set('aria-level', '2');
```

Comment:
@jelbourn - why not using existing aria-level (if specified) or fallback to default level of `2`? ... and warn user that the aria-level attribute is missing (if not specified).